### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1] Required information read as star.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11284]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/input
Required information for all mandatory input field is read out as ‘star’. Information as required for all the mandatory UIs should be communicated to the user

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use any input mandatory field. Required information for all mandatory input field is read out as ‘star’.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?

**Comment by nikolay-kolarov** [comment]
Hello,

The problem is that the star symbol is announced by the screen reader.
The require state is announced afterwards. So I assume this confused the tester.

In UI5 the star in the label is hidden:
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/9a931f6d-8bd2-47ed-8ac2-06dfc1e7a755)
So you can reuse the same technique.

Best Regards,
Nikolay
